### PR TITLE
Remote support

### PR DESCRIPTION
**Pull request by ole1986**
This PR should fix the issue #389 and #379

### CODE REVIEW

File: src/server/contentProvider.ts [hunk 1]
```diff
@@ -34,49 +34,52 @@ export class ContentProvider implements TextDocumentContentProvider {
...
+            });
 
         return `
                     <!DOCTYPE html>
-                    <head><style type="text/css"> html, body{ height:100%; width:100%; overflow:hidden; padding:0;margin:0; } </style>
-                    <titleCan I give a title</title>
-                    <script type="text/javascript">
-                        function start(){
+                    <head>
+                        <meta http-equiv="Content-Security-Policy" content="default-src * 'unsafe-inline' 'unsafe-eval'; script-src * 'unsafe-inline' 'unsafe-eval'; connect-src * 'unsafe-inline'; img-src * data: blob: 'unsafe-inline'; frame-src *; style-src * 'unsafe-inline';" />
```
Comment (mjbvz) [inline src/server/contentProvider.ts]: Please add a property content security policy. At this point, you may as well not even have one

Comment (ole1986) [reply src/server/contentProvider.ts]: @mjbvz Sorry? the meta tag is already defined. Anything else I need to define? HTTP headers?

Comment (mjbvz) [reply src/server/contentProvider.ts]: The current policy allows everything. This suppresses the warning in VS Code but does not bring any actually security benefits. 

See [docs like this](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) for help writing a proper csp.

Comment (ole1986) [reply src/server/contentProvider.ts]: I do know that this does not bring much security, but first I wanted to know why it does not display anything using macOS only...

Comment (ole1986) [reply src/server/contentProvider.ts]: For mac the webView port may always be different from the extensionHost port

**Pull request merged**